src/fides/api/service/connectors/sql_connector.py

```diff
@@ -146,6 +146,27 @@ def test_connection(self) -> Optional[ConnectionTestStatus]:
 
         return ConnectionTestStatus.succeeded
 
+    def execute_standalone_retrieval_query(
+        self, node: ExecutionNode, fields: List[str], filters: Dict[str, List[Any]]
+    ):
+        if not node.collection.custom_request_fields():
+            logger.error(
+                "Cannot call execute_standalone_retrieval_query on a collection without custom request fields"
+            )
+            return None
+
```

Comment:
not sure if we want to keep this check or if we want to allow other cases in which the standalone query could be useful